### ISSUE TITLE
Chapter 31 Orientation - Research and Implement Accessibility standards for video components

### ISSUE THREAD

**Issue by micahchiang** [department-of-veterans-affairs/va.gov-team#16808]
## Description
As an engineer, I want the components I build to be as accessible as possible, so the broadest spectrum of users possible can utilize my applications. Components that leverage video assets should be presented in an accessible way that includes the use of captions, the provision of a transcript, and the use of progressive enhancement. 

## Considerations
- Use of the `<video>` tag to embed video
- Using source sets to provide thumbnails at varying screen sizes
- Find a way to include transcripts

## Tasks
- [x] Research the use of `<video>` tags and replace the use of iframes with them
- [x] Research and implement source sets for screen sizes
- [x] Explore whether iFrames should be removed in favor of `<video>` tags

## Acceptance criteria
- [ ] ~~Transcript has been provided for the user~~
- [x] A list of next steps for iteration has been determined

**Next steps**  
- [ ] Varying screen sizes have been addressed
- [ ] Pull request has been made into `vets-website`

**Comment by jason-gcio** [comment]
It was determined that using `<video>` tags gives no measurable benefit and is not worth the lift of rebuilding these pages and creating new custom events.  We also can't deliver these faster than YT.

We can size the vides to be full width on mobile at a minimum.
